examples/convert_file_to_tlv.py

```diff
@@ -0,0 +1,74 @@
+# -----------------------------------------------------------------------------
+# Copyright (C) 2019-2020 Xinyu Ma, Susmit Shannigrahi
+#
+# This file is part of python-ndn.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+import logging
+import sys
+from ndn.utils import timestamp
+from ndn.app import NDNApp
+from ndn.encoding import Name, Component
+import multiprocessing
+import argparse
+
+#CPU_COUNT = multiprocessing.cpu_count() #they aren't used, just for testing
+#CPU_COUNT=1
+SEGMENT_SIZE = 8000 #this should be decided at runtime
```

Comment:
indeed, why don't you add this to the arg parser? seems easy enough?

Reply:
Updated.